Sources/Private/Model/Assets/AssetLibrary.swift

```diff
@@ -18,14 +18,17 @@ final class AssetLibrary: Codable, AnyInitializable, Sendable {
     var imageAssets = [String : ImageAsset]()
     var precompAssets = [String : PrecompAsset]()
 
-    while !container.isAtEnd {
-      let keyContainer = try containerForKeys.nestedContainer(keyedBy: PrecompAsset.CodingKeys.self)
-      if keyContainer.contains(.layers) {
-        let precompAsset = try container.decode(PrecompAsset.self)
+    while
+      !container.isAtEnd,
+      let keyContainer = try? containerForKeys.nestedContainer(keyedBy: PrecompAsset.CodingKeys.self)
+    {
+      if
+        keyContainer.contains(.layers),
+        let precompAsset = try? container.decode(PrecompAsset.self)
+      {
         decodedAssets[precompAsset.id] = precompAsset
         precompAssets[precompAsset.id] = precompAsset
-      } else {
-        let imageAsset = try container.decode(ImageAsset.self)
+      } else if let imageAsset = try? container.decode(ImageAsset.self) {
```

Comment:
If an image asset fails to parse for some reason, we can recover gracefully instead of throwing an error and rejecting the animation